docs/content/get-started/open-project.md

```diff
@@ -5,25 +5,23 @@ Launch the Unity Editor. It should automatically detect the project but if it do
 **Before you start, apply these two quick Unity bug fixes:**
 
 #### Shaders
-There is a bug in the current preview version of the [High Definition Render Pipeline](https://blogs.unity3d.com/2018/03/16/the-high-definition-render-pipeline-focused-on-visual-quality/), where shaders do not fully compile and appear visually darker than intended.
+There is a [known issue]({{urlRoot}}/known-issues) in the preview version of the [High Definition Render Pipeline](https://blogs.unity3d.com/2018/03/16/the-high-definition-render-pipeline-focused-on-visual-quality/), where shaders do not fully compile and appear visually darker than intended.
 
 There is a quick fix however:
 
 1. Open the FPS Starter Project in the Unity Editor located in `workers/unity`.
-2. In the Project panel, navigate to **Assets** > **Fps** > **Art** > **Materials**.
-3. Right click on `Source_Shaders` and press Reimport.
+2. In the Project window, navigate to **Assets** > **Fps** > **Art** > **Materials**.
+3. Right click on the `Source_Shaders` directory and select Reimport.
 
 <img src="{{assetRoot}}assets/shader-fix.png" style="margin: 0 auto; display: block;" />
 
 #### Bake Navmesh
-There is a bug where the Unity Editor does not import the navmesh for the `FPS-SimulatedPlayerCoordinator` correctly when opening a project for the first time. To fix this, you need to rebake the navmesh for this scene.
+There is a [known issue]({{urlRoot}}/known-issues) in the Unity Editor that stops it importing the navmesh for the `FPS-SimulatedPlayerCoordinator` correctly when you open the project for the first time. To fix this, you need to rebake the navmesh for this Scene. To do this:
```

Comment:
`stops it importing` sounds slightly off
maybe: 

There is a known issue in the Unity Editor regarding importing a navmesh. The navmesh for the `FPS-SimulatedPlayerCoordinator` is not imported correctly when opening the project for the first time.